h2o-py/tests/testdir_misc/pyunit_apply.py

```diff
@@ -114,6 +115,37 @@ def pyunit_apply_n_to_n_ops():
     test_ops(fr, pf, OPS_VEC_TO_VEC)
 
 
+def pyunit_apply_with_args():
+    fr = h2o_frame_fixture()
+    ref = fr.scale(center=False, scale=False).as_data_frame()
+
+    #vars
+    false = False
+    args = (False, False)
+    kwargs = dict(center=False, scale=False)
+    partial_args = (False,)
+    partial_kwargs = dict(scale=False)
+
+    to_test = dict(
+        scale_with_arg=lambda x: x.scale(False, False),
+        scale_with_kwarg=lambda x: x.scale(center=False, scale=False),
+        scale_with_argkwarg=lambda x: x.scale(False, scale=False),
+        scale_with_global_arg=(lambda x: x.scale(false, scale=false)) if not PY2 else None,
+        scale_with_args=(lambda x: x.scale(*args)) if not PY2 else None,
+        scale_with_kwargs=(lambda x: x.scale(**kwargs)) if not PY2 else None,
+        scale_with_partial_args=(lambda x: x.scale(False, *partial_args)) if not PY2 else None,
+        scale_with_partial_kwargs=(lambda x: x.scale(False, **partial_kwargs)) if not PY2 else None,
+        scale_with_args_and_kwargs=(lambda x: x.scale(*partial_args, **partial_kwargs)) if not PY2 else None,
+        scale_with_all_kind_args=(lambda x: x.scale(False, *partial_args, **partial_kwargs)) if not PY2 else None,  # strangely this works because our signature verification is not that strict, but it's fine... at least behaves as expected.
+    )
```

Comment:
It seems like all these tests are for the `CALL_METHOD` op. Can we also have tests for `CALL_FUNCTION_EX` (i.e. call a regular function, not a method).

Reply:
Also, in Py3.9 there will be added a new C function for calling python functions without any arguments. I don't know whether they'd be adding a new opcode for this, but it's possible. Thus, we should also add a test case for calling a function/method with no arguments.

Reply:
actually depending on Python version, those tests will use either `CALL_FUNCTION`, `CALL_METHOD`, `CALL_FUNCTION_KW` or `CALL_FUNCTION_EX`...
from 3.6, it's using `CALL_FUNCTION_EX` as soon as there is some unpacking needed (*args or **kwargs). 
Actually testing with `3.5`, not sure all of those are supported there.

Reply:
for the call without args, there are other tests in this file